minisom.py

```diff
@@ -538,6 +537,28 @@ def topographic_error(self, data):
         dxdy = hstack([diff(b2mu_x), diff(b2mu_y)])
         distance = norm(dxdy, axis=1)
         return (distance > t).mean()
+    
+    def topographic_error_hexagonal(self, data):
```

Comment:
If we have `topographic_error_hexagonal` we should also have `topographic_error_rectangular`. The naming convention about private methods applies also here.

Reply:
Updated